packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/react-native-aepcore",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Are we going to release core?

Reply:
https://github.com/adobe/aepsdk-react-native/blob/658dc507b8f3b85c8b7c0ae8d31c5bb2916ef893/packages/core/package.json#L3
The main branch shows 1.0.1 for core, I think it should be good to merge the PR. 

Btw, I released the core 1.0.1 about a month ago : https://www.npmjs.com/package/@adobe/react-native-aepcore/v/1.0.1